### PR TITLE
docs(zh): translate other zh docs

### PR DESCRIPTION
**Pull request by SoonIter**
## Summary

other zh docs

### CODE REVIEW

File: website/docs/en/guide/advanced/module-federation.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ import { PackageManagerTabs } from '@theme';
 
 <PackageManagerTabs command="add @module-federation/rsbuild-plugin -D" />
 
-Then add the plugin to the `rslib.config.ts` file:
+Then register the plugin to the `rslib.config.ts` file:
```
Comment (Timeless0911) [inline website/docs/en/guide/advanced/module-federation.mdx:31]:
```suggestion
Then register the plugin in `rslib.config.ts` file:
```

File: website/docs/en/guide/solution/nodejs.mdx [hunk 2]
```diff
@@ -54,7 +54,7 @@ export default defineConfig({
 
 Rslib set [target](/config/rsbuild/output#outputtarget) to `"node"` by default, which is different from the default target of Rsbuild.
 
-Rslib adjusts many configurations for Node.js. For example, [output.externals](/config/rsbuild/output#outputtarget) will exclude built-in Node.js modules, and [shims](/config/lib/shims) will add a shim for `import.meta.url` in CJS output by default.
+When target is valued as `"node"`, Rslib adjusts many configurations for Node.js. For example, [output.externals](/config/rsbuild/output#outputtarget) will exclude built-in Node.js modules, and [shims](/config/lib/shims) will add a shim for `import.meta.url` in CJS output by default.
```
Comment (Timeless0911) [inline website/docs/en/guide/solution/nodejs.mdx:57]:
```suggestion
When target is set to `"node"`, Rslib adjusts many configurations for Node.js. For example, [output.externals](/config/rsbuild/output#outputtarget) will exclude built-in Node.js modules, and [shims](/config/lib/shims) will add a shim for `import.meta.url` in CJS output by default.
```

File: website/docs/en/guide/solution/react.mdx [hunk 3]
```diff
@@ -79,7 +79,7 @@ export default defineConfig({
 - **Type**: `string`
 - **Default**: `'react'`
 
-When `runtime` is `'automatic'`, you can specify the import path of the JSX transform through `importSource`.
+When `runtime` is valued as `'automatic'`, you can specify the import path of the JSX transform through `importSource`.
```
Comment (Timeless0911) [inline website/docs/en/guide/solution/react.mdx:82]:
```suggestion
When `runtime` is set to `'automatic'`, you can specify the import path of the JSX transform through `importSource`.
```

File: website/docs/zh/guide/advanced/dts.mdx [hunk 4]
```diff
@@ -1 +1,65 @@
...
+本章介绍什么是 TypeScript 声明文件（DTS）以及如何在 Rslib 中生成 DTS 文件。
+
+## 什么是 DTS
+
+TypeScript 声明文件 (DTS) 提供 JavaScript 代码的类型信息。 DTS 文件通常具有 `.d.ts` 扩展名。它们允许 TypeScript 编译器理解 JavaScript 代码的类型结构，从而实现以下功能：
+
+1. **类型检查**: 为 JavaScript 代码提供类型信息，帮助开发人员在编译时捕获潜在的类型错误。
+2. **代码补全**: 增强代码编辑器功能，例如自动完成和代码导航。
+3. **文档生成**: 生成 JavaScript 代码文档，提供更好的开发体验。
+4. **IDE 支持**: 改善 Visual Studio Code、WebStorm 等 IDE 中的开发人员体验.
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/dts.mdx:12]:
```suggestion
4. **IDE 支持**: 改善 Visual Studio Code、WebStorm 等 IDE 中的开发者体验。
```

File: website/docs/zh/guide/advanced/dts.mdx [hunk 5]
```diff
@@ -1 +1,65 @@
...
+
+Bundle DTS 涉及将多个 TypeScript 声明文件 bundle 到一个声明文件中。
+
+- **优势:**
+
+  - **简化管理**: 简化类型文件的管理和引用。
+  - **容易分发**: 减少用户使用库时需要处理的文件数量。
+
+- **劣势:**
+  - **生成复杂**: 在大型项目中，生成和维护单个 bundle 文件可能会变得复杂.
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/dts.mdx:27]:
```suggestion
  - **生成复杂**: 在大型项目中，生成和维护单个 bundle 文件可能会变得复杂。
```

File: website/docs/zh/guide/advanced/dts.mdx [hunk 6]
```diff
@@ -1 +1,65 @@
...
+- **优势:**
+
+  - **模块化**: 每个模块都有自己的类型定义，使维护和调试更容易。
+  - **灵活**: 适合大型项目，避免单个文件的复杂性。
+
+- **劣势:**
+  - **多文件**: 用户在使用该库时可能需要处理多个声明文件。
+  - **管理复杂**: 可能需要额外的配置才能正确引用所有文件。
+
+## 怎么在 Rslib 中生成 DTS
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/dts.mdx:43]:
```suggestion
## 如何在 Rslib 中生成 DTS
```

File: website/docs/zh/guide/advanced/dts.mdx [hunk 7]
```diff
@@ -1 +1,65 @@
...
+
+Rslib 默认使用 [TypeScript Compiler API](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API) 生成 Bundleless DTS， [API Extractor](https://api-extractor.com/)
+
+如果你想生成 Bundleless DTS，可以：
+
+- 设置 `dts: true` 或者 `dts: { bundle: false }` 在 Rslib 配置文件。
+
+如果你想生成 Bundle DTS，可以：
+
+1. 安装 `@microsoft/api-extractor` 作为 `dev`, 这是用于 bundle DTS 文件的底层工具。
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/dts.mdx:53]:
```suggestion
1. 安装 `@microsoft/api-extractor` 作为 `devDdevDependencies`, 这是用于 bundle DTS 文件的底层工具。
```

File: website/docs/zh/guide/advanced/dts.mdx [hunk 8]
```diff
@@ -1 +1,65 @@
...
+
+import { PackageManagerTabs } from '@theme';
+
+<PackageManagerTabs command="add @microsoft/api-extractor -D" />
+
+2. 设置 `dts: { bundle: true }` 在 Rslib 配置文件中。
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/dts.mdx:59]:
```suggestion
2. 在 Rslib 配置文件中设置 `dts: { bundle: true }`。
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 9]
```diff
@@ -1 +1,330 @@
...
+
+## 使用场景
+
+模块联邦有一些典型的使用场景，包括：
+
+- 允许独立应用程序（微前端架构中称为“微前端”）共享模块，而无需重新编译整个应用。
+- 不同的团队处理同一应用程序的不同部分，而无需重新编译整个应用程序。
+- 运行时中在应用间动态加载和共享代码。
+
+模块联盟可以帮助你:
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:16]:
```suggestion
模块联邦可以帮助你:
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 10]
```diff
@@ -1 +1,330 @@
...
+
+## 快速开始
+
+首先安装 [Module Federation Rsbuild Plugin](https://www.npmjs.com/package/@module-federation/rsbuild-plugin).
+
+import { PackageManagerTabs } from '@theme';
+
+<PackageManagerTabs command="add @module-federation/rsbuild-plugin -D" />
+
+然后在 `rslib.config.ts` 注册插件:
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:31]:
```suggestion
然后在 `rslib.config.ts` 中注册插件:
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 11]
```diff
@@ -1 +1,330 @@
...
+
+但是，如果你希望此 Rslib Module 同时消费其他生产者，请不要使用构建配置 `remote` 参数，因为在其他格式下，这可能会导致错误，请参考下面使用 Module Federation 运行时的示例
+
+## 开发 MF 远程模块
+
+### 使用宿主应用
+
+Rslib 支持宿主应用和 Rslib 模块联邦项目同时开发。
+
+#### 1. 启动库中的 MF `dev`
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:97]:
```suggestion
#### 1. 启动库的 MF 开发模式
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 12]
```diff
@@ -1 +1,330 @@
...
+});
+```
+
+然后通过 `rsbuild dev` 启动 Host app。
+
+### 使用 Storybook
+
+Rslib 支持使用 Storybook 开发 Module Federation Rslib 项目。
+
+#### 1. 启动库的 MF `dev`
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:154]:
```suggestion
#### 1. 启动库的 MF 开发模式
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 13]
```diff
@@ -1 +1,330 @@
...
+```json title="package.json"
+{
+  "scripts": {
+    "dev": "rslib mf dev"
+  }
+}
+```
+
+然后运行 `dev` 命令即可启动 Module Federation 开发模式，可被 Host App 消费，
+同时具有热模块更换（HMR）功能。
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:110]:
```suggestion
同时具有模块热更新（HMR）功能。
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 14]
```diff
@@ -1 +1,330 @@
...
+```json title="package.json"
+{
+  "scripts": {
+    "dev": "rslib mf dev"
+  }
+}
+```
+
+然后运行 `dev` 命令即可启动 Module Federation 开发模式，可被 Storybook 消费，
+同时具有热模块更换（HMR）功能。
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:167]:
```suggestion
同时具有模块热更新（HMR）功能。
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 15]
```diff
@@ -1 +1,330 @@
...
+        },
+      },
+    },
+  ],
+};
+
+export default config;
+```
+
+#### 3. 用远程模块 编写 stories
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:229]:
```suggestion
#### 3. 用远程模块编写 stories
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 16]
```diff
@@ -1 +1,330 @@
...
+
+export default {
+  title: 'App Component',
+  component: Component,
+};
+
+export const Primary = {};
+```
+
+#### 4. 在 `tsconfig.json` 中添加 Module Federation 类型和 stories 文件。
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx]:
```suggestion
#### 4. 在 `tsconfig.json` 中添加 Module Federation 类型和 stories 文件
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 17]
```diff
@@ -1 +1,330 @@
...
+}
+```
+
+#### 5. 启动 Storybook app
+
+大功告成，启动 Storybook `npx storybook dev`。
+
+## 使用其他模块联合模块
+
+由于 Rslib 中有多种格式，如果在构建时配置 `remote` 参数来消耗其他模块，则可能无法在所有格式下正常工作。建议通过以下方式访问 [Module Federation Runtime](https://module-federation.io/guide/basic/runtime.html)
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:268]:
```suggestion
由于 Rslib 中有多种格式，如果在构建时配置 `remote` 参数来消耗其他模块，则可能无法在所有格式下正常工作。建议通过以下方式访问 [Module Federation Runtime](https://module-federation.io/zh/guide/basic/runtime.html)
```

File: website/docs/zh/guide/advanced/module-federation.mdx [hunk 18]
```diff
@@ -1 +1,330 @@
...
+  );
+};
+```
+
+这确保了模块可以按预期以多种格式加载。
+
+## FAQs
+
+如果 Rslib 生产者是用 build 构建的， 这意味着生产者中的 `process.env.NODE_ENV` 是 `production` 。如果这时消费者是使用的开发模式启动，由于模块联邦默认使用共享的加载策略，可能会有 react 和 react-dom 加载模式不一致的问题 (比如 react 在 development mode, react-dom 在 production mode)。
+你可以在消费者设置 [shareStrategy](https://module-federation.io/configure/sharestrategy) 来解决这个问题，但是确保你已经完全理解了这个配置。
```
Comment (Timeless0911) [inline website/docs/zh/guide/advanced/module-federation.mdx:313]:
```suggestion
你可以在消费者设置 [shareStrategy](https://module-federation.io/zh/configure/sharestrategy) 来解决这个问题，这需要你确保已经完全理解了这个配置。
```

File: website/docs/zh/guide/solution/index.mdx [hunk 19]
```diff
@@ -1 +1,25 @@
 # 解决方案
+
+在本章中，将介绍如何使用 Rslib 开发浏览器和 Node.js 的库。我们还将介绍如何为不同的 UI 框架创建库。
+
+## Browser Target
+
+开发在浏览器中运行的库时，可以将其打包为 [ESM](/guide/basic/output-format#esm--cjs) 和 [CJS](/guide/basic/output-format#esm--cjs) 格式，用于与 app 的 bundler 集成。将包 [conditional-exports](https://nodejs.org/api/packages.html#conditional-exports) 配置为 ESM 输出可以更好地进行 treeshaking。此外，你可以创建 [UMD](/guide/basic/output-format#umd) 格式输出以供浏览器直接使用，甚至可以生成 [模块联邦](/guide/advanced/module-federation) 格式以供其他应用程序动态加载。根据目标浏览器支持配置 [Browserslist](https://rsbuild.dev/guide/advanced/browserslist)以确定输出的降级语法，或添加 [polyfill](/guide/advanced/output-compatibility) 用于兼容 API。
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/index.mdx:7]:
```suggestion
开发在浏览器中运行的库时，可以将其打包为 [ESM](/guide/basic/output-format#esm--cjs) 和 [CJS](/guide/basic/output-format#esm--cjs) 格式，用于与 app 的 bundler 集成。将包 [conditional-exports](https://nodejs.org/api/packages.html#conditional-exports) 配置为 ESM 输出可以更好地进行 treeshaking。此外，你可以创建 [UMD](/guide/basic/output-format#umd) 格式输出以供浏览器直接使用，甚至可以生成 [模块联邦](/guide/advanced/module-federation) 格式以供其他应用程序动态加载。根据目标浏览器支持配置 [Browserslist](https://rsbuild.dev/zh/guide/advanced/browserslist)以确定输出的降级语法，或添加 [polyfill](/guide/advanced/output-compatibility) 用于兼容 API。
```

File: website/docs/zh/guide/solution/index.mdx [hunk 20]
```diff
@@ -1 +1,25 @@
...
+
+发布到 npm 时，你可以选择不压缩代码或[压缩代码](/config/rsbuild/output#outputminify)，同时提供[sourcemap](/config/rsbuild/output#outputsourcema) 以增强库使用者的调试体验。样式上，可以使用 [CSS](/guide/advanced/css) 或 [CSS 预处理器](/guide/advanced/css#preprocessors)，如 Sass、Less 或 Stylus 等，或使用 [PostCSS](/guide/advanced/css#postcss) 用于 CSS 后处理。 [Tailwind CSS](/guide/advanced/css#tailwind-css) 等工具也可以帮助你构建样式。或使用 [CSS 模块](/guide/advanced/css#css-modules)。
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/index.mdx:9]:
```suggestion
发布到 npm 时，你可以选择不压缩代码或[压缩代码](/config/rsbuild/output#outputminify)，同时提供[sourcemap](/config/rsbuild/output#outputsourcema) 以增强库使用者的调试体验。样式上，可以使用 [CSS](/guide/advanced/css) 或 [CSS 预处理器](/guide/advanced/css#preprocessors)，如 Sass、Less 或 Stylus 等，或使用 [PostCSS](/guide/advanced/css#postcss) 用于 CSS 后处理。 [Tailwind CSS](/guide/advanced/css#tailwind-css) 等工具也可以帮助你构建样式。或使用 [CSS Modules](/guide/advanced/css#css-modules)。
```

File: website/docs/zh/guide/solution/index.mdx [hunk 21]
```diff
@@ -1 +1,25 @@
...
+
+在资源管理方面，Rslib 处理代码中使用的[静态资源](/guide/advanced/assets)，例如 SVG 和 PNG 文件。你还可以构建[React](/guide/solution/react)、[Preact](https://github.com/web-infra-dev/rslib/tree/main/examples/preact-component-的组件库Bundle-false) 或其他框架，使用 [Storybook](/guide/advanced/storybook) 进行 UI 组件开发和测试。
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/index.mdx:11]:
```suggestion
在资源处理方面，Rslib 处理代码中使用的[静态资源](/guide/advanced/assets)，例如 SVG 和 PNG 文件。你还可以构建[React](/guide/solution/react)、[Preact](https://github.com/web-infra-dev/rslib/tree/main/examples/preact-component-bundle-false) 或其他框架，使用 [Storybook](/guide/advanced/storybook) 进行 UI 组件开发和测试。
```

File: website/docs/zh/guide/solution/index.mdx [hunk 22]
```diff
@@ -1 +1,25 @@
...
+
+参考本章的解决方案，了解如何使用 Rslib 开发一个在浏览器中使用的库，给多种框架使用。
+
+{/* TODO: Clarify default behavior */}
+{/* ### Default Behavior */}
+
+## Node.js Target
+
+Rslib [target](/config/rsbuild/output#outputtarget) 默认值为 `"node"` 来开发 Node.js 中使用的库。
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/index.mdx:20]:
```suggestion
Rslib 默认将 [target](/config/rsbuild/output#outputtarget) 设置为 `"node"`，以方便开发 Node.js 中使用的库。
```

File: website/docs/zh/guide/solution/react.mdx [hunk 23]
```diff
@@ -1 +1,113 @@
...
+  }}
+/>
+
+然后，当提示 "Select template" 选择 `React`。
+
+## 在现有项目中使用 Rslib
+
+开发 React 组件，需要设置 [target](/config/rsbuild/output#outputtarget) 为 `"web"` 在 `rslib.config.ts` 中。 这一点至关重要，因为 Rslib 默认将 `target` 设置为 `"node"`，这与 Rsbuild 的 target 默认值不同。
+
+要编译 React（JSX 和 TSX），你需要注册 Rsbuild [React 插件](https://rsbuild.dev/plugins/list/plugin-react)。该插件将自动添加 React 构建所需的配置。
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/react.mdx:26]:
```suggestion
要编译 React（JSX 和 TSX），你需要注册 Rsbuild [React 插件](https://rsbuild.dev/zh/plugins/list/plugin-react)。该插件将自动添加 React 构建所需的配置。
```

File: website/docs/zh/guide/solution/react.mdx [hunk 24]
```diff
@@ -1 +1,113 @@
...
+## JSX Transform
+
+- **类型**: `'automatic' | 'classic'`
+- **默认值**: `'automatic'`
+
+React 引入了一个 [新的 JSX transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) 在版本 17 中。这个新的 transform 在使用 JSX 时无需导入 `React`。
+
+默认情况下，Rsbuild 使用新的 JSX 转换，即“ `runtime: 'automatic'`。需要 React `16.14.0` 或更高版本。 `peerDependencies` 中应声明 `"react": ">=16.14.0"`。
+
+要更改 JSX transform，可以传递 [swcReactOptions](https://rsbuild.dev/plugins/list/plugin-react#swcreactoptionsruntime) 给 React plugin. 比如要使用 classic runtime 时:
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/react.mdx:54]:
```suggestion
要更改 JSX transform，可以传递 [swcReactOptions](https://rsbuild.dev/zh/plugins/list/plugin-react#swcreactoptionsruntime) 给 React plugin. 比如要使用 classic runtime 时:
```

File: website/docs/zh/guide/solution/react.mdx [hunk 25]
```diff
@@ -1 +1,113 @@
...
+  ],
+});
+```
+
+{/* TODO */}
+{/* ## SVGR */}
+
+## 进一步了解
+
+- [Rsbuild React Plugin](https://rsbuild.dev/plugins/list/plugin-react#swcreactoptionsruntime)
```
Comment (Timeless0911) [inline website/docs/zh/guide/solution/react.mdx:112]:
```suggestion
- [Rsbuild React Plugin](https://rsbuild.dev/zh/plugins/list/plugin-react#swcreactoptionsruntime)
```

File: website/docs/zh/guide/start/glossary.mdx [hunk 26]
```diff
@@ -1 +1,34 @@
...
+
+CJS 代表 [CommonJS](https://nodejs.org/api/modules.html#modules-commonjs-modules)，<CJS />
+
+## UMD
+
+<UMD />
+
+## Bundleless
+
+Bundleless 是指一种开发方法，它避免了将多个 JavaScript /TypeScript 文件捆绑到单个或更少的输出文件中，然后再将其提供给客户端的传统做法。相反，它的目标是直接为各个模块提供服务。
```
Comment (Timeless0911) [inline website/docs/zh/guide/start/glossary.mdx:22]:
```suggestion
Bundleless 是指一种开发模式，它避免了将多个 JavaScript /TypeScript 文件捆绑到单个或更少的输出文件中，然后再将其提供给客户端的传统做法。相反，它的目标是直接为各个模块提供服务。
```

File: website/docs/zh/guide/start/glossary.mdx [hunk 27]
```diff
@@ -1 +1,34 @@
...
+
+## DTS
+
+DTS代表[TypeScript声明文件](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html)，为JavaScript代码提供类型信息。
```
Comment (Timeless0911) [inline website/docs/zh/guide/start/glossary.mdx:26]:
```suggestion
DTS 代表 [TypeScript 类型声明文件](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html)，为 JavaScript 代码提供类型信息。
```

File: website/docs/zh/guide/start/glossary.mdx [hunk 28]
```diff
@@ -1 +1,34 @@
...
+
+## 模块联邦
+
+<MF />
+
+## 更多
+
+访问 [Rsbuild-术语表](https://rsbuild.dev/guide/start/glossary) 和 [Rspack-术语表](https://rspack.dev/misc/glossary) 中的更多名词解释。
```
Comment (Timeless0911) [inline website/docs/zh/guide/start/glossary.mdx:34]:
```suggestion
访问 [Rsbuild - 术语表](https://rsbuild.dev/zh/guide/start/glossary) 和 [Rspack - 术语表](https://rspack.dev/zh/misc/glossary) 中的更多名词解释。
```

File: website/docs/zh/guide/start/npm-packages.mdx [hunk 29]
```diff
@@ -1 +1,30 @@
...
+Rslib 核心包，提供 CLI 命令和基于 Rsbuild 的构建功能。
+
+- [npm](https://npmjs.com/package/@rslib/core)
+- [Source Code](https://github.com/web-infra-dev/rslib/tree/main/packages/core)
+
+## rsbuild-plugin-dts
+
+![](https://img.shields.io/npm/v/rsbuild-plugin-dts?style=flat-square&colorA=564341&colorB=F8F5FF)
+
+支持为 TypeScript 发出声明文件的 Rsbuild 插件。
```
Comment (Timeless0911) [inline website/docs/zh/guide/start/npm-packages.mdx:18]:
```suggestion
支持为 TypeScript 生成类型声明文件的 Rsbuild 插件。
```

File: website/docs/zh/guide/start/quick-start.mdx [hunk 30]
```diff
@@ -1 +1,133 @@
...
+| Template                     | Description                  |
+| ---------------------------- | ---------------------------- |
+| Node.js dual ESM/CJS package | Node.js dual ESM/CJS package |
+| Node.js pure ESM package     | Node.js pure ESM package     |
+| React                        | React component library      |
+
+每个模板都支持 JavaScript 和 TypeScript，以及可选的开发工具、格式化程序和 linter。
+
+:::info
+我们正在努力为更多框架提供模板 (比如 Vue).
```
Comment (Timeless0911) [inline website/docs/zh/guide/start/quick-start.mdx:54]:
```suggestion
我们正在努力为更多框架提供模板 (比如 Vue)。
```

**Comment by Timeless0911** [comment]
I think we can remove banner in zh docs in this PR.

File: website/docs/en/guide/solution/react.mdx (hunk 3 above)
Comment (SoonIter) [reply website/docs/en/guide/solution/react.mdx:82]: mark

File: website/docs/en/guide/advanced/module-federation.mdx (hunk 1 above)
Comment (Timeless0911) [reply website/docs/en/guide/advanced/module-federation.mdx:31]: Both "register the plugin to the rslib.config.ts file" and "register the plugin in rslib.config.ts file" are grammatically correct, but there's a subtle difference in meaning that might make one more appropriate depending on the context.
"Register the plugin to the rslib.config.ts file" implies a more direct action of linking the plugin to the file. It suggests the plugin is being added as a component or element within the file.
"Register the plugin in the rslib.config.ts file" implies a broader sense of inclusion within the file. It suggests the plugin is being recognized and acknowledged within the file's configuration.
Ultimately, the best choice depends on the specific way the plugin is being registered and the intended meaning.
If you're unsure, "register the plugin in the rslib.config.ts file" is a safer and more general option.

File: website/docs/zh/guide/advanced/dts.mdx (hunk 7 above)
Comment (SoonIter) [reply website/docs/zh/guide/advanced/dts.mdx:53]: mark

File: website/docs/zh/guide/advanced/dts.mdx (hunk 5 above)
Comment (SoonIter) [reply website/docs/zh/guide/advanced/dts.mdx:27]: mark

File: website/docs/zh/guide/advanced/dts.mdx (hunk 6 above)
Comment (SoonIter) [reply website/docs/zh/guide/advanced/dts.mdx:43]: mark

File: website/docs/zh/guide/advanced/dts.mdx (hunk 8 above)
Comment (SoonIter) [reply website/docs/zh/guide/advanced/dts.mdx:59]: mark

File: website/docs/zh/guide/advanced/module-federation.mdx (hunk 9 above)
Comment (SoonIter) [reply website/docs/zh/guide/advanced/module-federation.mdx:16]: mark

File: website/docs/zh/guide/advanced/module-federation.mdx (hunk 11 above)
Comment (SoonIter) [reply website/docs/zh/guide/advanced/module-federation.mdx:97]: no, it is a command, not a mode

File: website/docs/zh/guide/start/quick-start.mdx (hunk 30 above)
Comment (SoonIter) [reply website/docs/zh/guide/start/quick-start.mdx:54]: mark

File: website/docs/zh/guide/start/npm-packages.mdx (hunk 29 above)
Comment (SoonIter) [reply website/docs/zh/guide/start/npm-packages.mdx:18]: mark

File: website/docs/zh/guide/advanced/module-federation.mdx (hunk 11 above)
Comment (Timeless0911) [reply website/docs/zh/guide/advanced/module-federation.mdx]: if so, `启动 mf dev 命令` is better

Comment (SoonIter) [reply website/docs/zh/guide/advanced/module-federation.mdx]: 启动库的 `rslib mf dev`

File: website/docs/zh/guide/solution/react.mdx [hunk 31]
```diff
@@ -1 +1,113 @@
...
+```
+
+## JSX Transform
+
+- **类型**: `'automatic' | 'classic'`
+- **默认值**: `'automatic'`
+
+React 引入了一个 [新的 JSX transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) 在版本 17 中。这个新的 transform 在使用 JSX 时无需导入 `React`。
+
+默认情况下，Rsbuild 使用新的 JSX 转换，即“ `runtime: 'automatic'`。需要 React `16.14.0` 或更高版本。 `peerDependencies` 中应声明 `"react": ">=16.14.0"`。
```
Comment (SoonIter) [inline website/docs/zh/guide/solution/react.mdx:52]:
```suggestion
默认情况下，Rsbuild 使用新的 JSX 转换，即 `runtime: 'automatic'`。需要 React `16.14.0` 或更高版本。 `peerDependencies` 中应声明 `"react": ">=16.14.0"`。
```

**Review by Timeless0911 — APPROVED** [review]

**Pull request merged**